agent/cmd/start.go

```diff
@@ -37,6 +37,7 @@ const (
 	flagDeregistrationHandler    = "deregistration-handler"
 	flagEventsRateLimit          = "events-rate-limit"
 	flagEventsBurstLimit         = "events-burst-limit"
+	flagKeepaliveHandler         = "keepalive-handler"
```

Comment:
This is going to sound like such a nitpick, but wouldn't `keepalive-handlers` make more sense since a user can define multiple handlers? (ex. `--subscriptions` can define 0 to many subscriptions)